Author: Ingo Weyrich <heckflosse67@gmx.de>
Description Fix segfault on close when started with a filename as argument
Source: https://github.com/Beep6581/RawTherapee/commit/e7a4ae3c9fa987d625b537573fd811a9cb5d2fb8.patch
Last-update: 2020-05-30
--- a/rtgui/rtwindow.cc
+++ b/rtgui/rtwindow.cc
@@ -99,6 +99,8 @@ RTWindow::RTWindow ()
     , bpanel (nullptr)
     , splash (nullptr)
     , btn_fullscreen (nullptr)
+    , iFullscreen (nullptr)
+    , iFullscreen_exit (nullptr)
     , epanel (nullptr)
     , fpanel (nullptr)
 {
